pkg/formatters/formatters.go

```diff
@@ -0,0 +1,7 @@
+package formatters
+
+import "github.com/spf13/cobra"
+
+func CompleteOutputFormat(*cobra.Command, []string, string) ([]string, cobra.ShellCompDirective) {
```

Comment:
Maybe the really basic static ones should be implemented like this, ie:

- kinds
- plugin names
- output formats

Plugin names can change according to the version of fluent-bit is running on a pipeline though...